server/api_server/api_server_start.py

```
@@ -848,15 +1215,37 @@ def api_get_events_totals():
 
 @app.route('/mcp/sse/events/recent', methods=['GET', 'POST'])
 @app.route('/events/recent', methods=['GET'])
-def api_events_default_24h():
-    return api_events_recent(24)  # Reuse handler
+@validate_request(
+    operation_id="get_recent_events",
+    summary="Get Recent Events",
+    description="Get recent events from the system.",
+    request_model=RecentEventsRequest
+)
+def api_events_default_24h(payload=None):
```

Ruff (0.14.11)

Unused function argument: `payload`

(ARG001)